### ISSUE TITLE
Walking Speed can not be set under 1km/hour 

### ISSUE THREAD

**Issue by E10116** [AnthonyOSX/GO-Bot#173]
GO Bot v0.0.5
In case I want to stay stable on a position, I cloud set the Walking Speed option on Configuration = 0.
It is useful in case you want to stay around popular location full of pokestops with lure modules.

**Comment by jodriscoll** [comment]
Great idea!

**Comment by Mod101** [comment]
agreed, we need this option, only problem will be, what happens when you run out of pokeballs? lets say you dont get enough at the stops... will bot hunt for pokeballs, and come back to the same spot to grind, maybe..
